lib/tasks/download-plugin-readmes.js

```diff
@@ -0,0 +1,58 @@
+// Download readmes form plugin repos on GitHub
+
+// This is NOT an automatic process: use your own judgement about which changes
+// to a plugin's readme are important and need to be included in the docs.
+// Consider naming (remove `Swup ` prefix from title) and keep the frontmatter.
+
```

Comment:
Maybe this warning should be logged to the console when running `npm run plugins:readme`?

Reply:
Good idea 🥯